src/test/java/teammates/test/cases/ui/browsertests/AdminAccountManagementPageUiTest.java

```diff
@@ -110,6 +110,14 @@ public void testDeleteInstructorAccountAction(){
     private void loginToAdminAccountsManagementPage() {
         accountsPageUrl = createUrl(Const.ActionURIs.ADMIN_ACCOUNT_MANAGEMENT_PAGE + "?all=true");
         accountsPage = loginAdminToPageForAdminUiTests(browser, accountsPageUrl, AdminAccountManagementPage.class);
+
+        boolean isDevEnvironment = Boolean.parseBoolean(System.getProperty("isDevEnvironment"));
+
+        // On non-dev environments, page can take a long time to load
+        if (!isDevEnvironment) {
```

Comment:
It should be fine if this is done without checking for the dev environment too right?